plugins/logs/plugin_test.go

```diff
@@ -482,6 +482,106 @@ func TestPluginRequeBufferPreserved(t *testing.T) {
 	}
 }
 
+func TestPluginRateLimitInt(t *testing.T) {
+	ctx := context.Background()
+
+	numDecisions := 1 // 1 decision per second
+
+	fixture := newTestFixture(t, func(c *Config) {
+		limit := float64(numDecisions)
+		c.Reporting.MaxDecisionsPerSecond = &limit
+	}, func(c *Config) {
+		limit := int64(300)
+		c.Reporting.UploadSizeLimitBytes = &limit
+	})
+	defer fixture.server.stop()
+
+	var input interface{} = map[string]interface{}{"method": "GET"}
+	var result interface{} = false
+
+	_ = fixture.plugin.Log(ctx, logServerInfo("abc", input, result))
+	_ = fixture.plugin.Log(ctx, logServerInfo("def", input, result))
+
+	_ = fixture.plugin.Log(ctx, logServerInfo("ghi", input, result)) // should not be added to buffer as rate limit exceeded
```

Comment:
Would the second one be there?

Reply:
Ignore me. The second one is buffered, the first one was sent?

Reply:
Updated the test to make it more clear which event is buffered.